client/components/share/nextdoor-share-preview/index.jsx

```diff
@@ -0,0 +1,39 @@
+import { NextdoorPreviews } from '@automattic/social-previews';
+import { PureComponent } from 'react';
+import { decodeEntities } from 'calypso/lib/formatting';
+
+export class NextdoorSharePreview extends PureComponent {
```

Comment:
Is this for consistency or is there an advantage to this over a functional component?

Reply:
Just for consistency.